OpenDreamClient/Interface/InterfaceMacro.cs

```diff
@@ -293,14 +292,14 @@ public sealed class InterfaceMacro : InterfaceElement {
             return;
         }
 
-        if (_entitySystemManager.TryGetEntitySystem(out DreamCommandSystem? commandSystem)) {
+
             string? keyName = ParsedKeybind.KeyToKeyName(args.Key);
             if (keyName == null)
                 return;
             string command = Command.Replace("[[*]]", keyName);
```

Comment:
Dedent these lines now that there's no `if` surrounding them
```suggestion
        string? keyName = ParsedKeybind.KeyToKeyName(args.Key);
        if (keyName == null)
            return;
        string command = Command.Replace("[[*]]", keyName);
```